scopesim/utils.py

```diff
@@ -251,14 +249,15 @@ def add_SED_to_scopesim(file_in, file_out=None, wave_units="um"):
 
     """
 
-    file_name, file_ext = os.path.basename(file_in).split(".")
+    path = Path(file_in)
 
     if file_out is None:
-        if "SED_" not in file_name:
-            file_out = rc.__data_dir__ + "SED_" + file_name + ".dat"
-        else: file_out = rc.__data_dir__ + file_name + ".dat"
+        if "SED_" not in path.name:
+            file_out = rc.__data_dir__ + f"SED_{path.name}.dat"
+        else:
+            file_out = rc.__data_dir__ + f"{path.name}.dat"
 
-    if file_ext.lower() in "fits":
+    if path.suffix.lower() in "fits":
```

Comment:
```suggestion
    if path.suffix.lower() == ".fits":
```
There was a bug here, because `in "fits"` would check whether the suffix is one of the letters `f`, `i`, `t`, or `s`, and this change introduced another one, because `Path("myfile.fits").suffix` returns `".fits"`, with the colon.